### PR TITLE
[DNM]Ci conventions

### PR DESCRIPTION
**Pull request by raukadah**
As a pull request owner and reviewers, I checked that:
- [ ] Appropriate testing is done and actually running

### CODE REVIEW

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -49,5 +49,3 @@
       - cifmw-molecule-tempest
       - cifmw-molecule-test_deps
     name: openstack-k8s-operators/ci-framework
-    templates:
```
Comment (cjeanner) [inline zuul.d/projects.yaml:52]: you probably didn't edit the ci/templates/project.yml.j2 when you added the templates support for the project?

File: config/deployment/.edpm_ci.yml.swp
Comment (cjeanner) [inline config/deployment/.edpm_ci.yml.swp:1]: please remove that file

**Comment by raukadah** [comment]
closing in favor of https://github.com/openstack-k8s-operators/ci-framework/pull/580

**Pull request closed without merging**